test/ably/restchannelpublish_test.py

```diff
@@ -78,43 +75,40 @@ def test_publish_various_datatypes_text(self):
                 message_contents["publish6"],
                 msg="Expect publish6 to be expected JSONObject")
 
-   def test_publish_various_datatypes_binary(self):
-       publish1 = TestRestChannelPublish.ably_binary.channels.publish1
+    def test_publish_various_datatypes_binary(self):
+        publish1 = TestRestChannelPublish.ably_binary.channels.publish1
 
-       publish1.publish("publish0", True)
-       publish1.publish("publish1", 24)
-       publish1.publish("publish2", 24.234)
-       publish1.publish("publish3", "This is a string message payload")
-       publish1.publish("publish4", bytearray("This is a byte[] message payload", "utf_8"))
-       publish1.publish("publish5", {"test": "This is a JSONObject message payload"})
-       publish1.publish("publish6", ["This is a JSONArray message payload"])
+        publish1.publish("publish0", True)
```

Comment:
Ignore this test, didn't change anything just fixed broken indentation. They will be rewritten at some point.